docs/source/metrics.rst

```diff
@@ -334,6 +334,8 @@ Complete list of metrics
     Rouge
     RougeL
     RougeN
+    InceptionScore
+    InceptionModel
```

Comment:
It is not a metric, why do you put it in the list of metric ?

Reply:
Where should I put it?